experimental/packages/opentelemetry-instrumentation-http/src/utils.ts

```diff
@@ -96,10 +96,10 @@ export const getAbsoluteUrl = (
   const reqUrlObject = requestUrl || {};
   const protocol = reqUrlObject.protocol || fallbackProtocol;
   const port = (reqUrlObject.port || '').toString();
-  const path = reqUrlObject.path || '/';
+  let path = reqUrlObject.path || '/';
   let host =
     reqUrlObject.host || reqUrlObject.hostname || headers.host || 'localhost';
-
+  const STR_REDACTED = 'REDACTED';
```

Comment:
Since this is a constant, does it make sense to hoist it out of this function to the top level of the file?